scripts/build_for_develop.bat

```diff
@@ -19,25 +24,34 @@ for /f "delims=" %%a in ('%CONDA_PREFIX%\python.exe -c "import numpy; print(nump
 set PYTHON_INC=
 for /f "delims=" %%a in ('%CONDA_PREFIX%\python.exe -c "import distutils.sysconfig as sc; print(sc.get_python_inc())"') do @set PYTHON_INC=%%a 
 
+if defined USE_GTEST (
+    set "_GTEST_INCLUDE_DIR=%CONDA_PREFIX%\Library\include"
+    set "_GTEST_LIB_DIR=%CONDA_PREFIX%\Library\lib"
+) else (
+    set "_GTEST_INCLUDE_DIR="
+    set "_GTEST_LIB_DIR="
+)
 cmake -G Ninja ^
-    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_BUILD_TYPE=Debug ^
```

Comment:
Let's keep this as Release and add a comment that it may not work if changed to Debug because of C++ std lib problems.

Reply:
Sure. Will push.